addons/mail/models/mail_notification.py

```diff
@@ -16,6 +16,7 @@ class MailNotification(models.Model):
     _description = 'Message Notifications'
 
     # origin
+    author_id = fields.Many2one('res.partner', "Author", ondelete='set null')
```

Comment:
Don't mix ' and " if possible ^^

Reply:
Don't know if there are guidelines ... I just avoid mixin simple and double quotes, makes my brain freeze.